ctapipe/io/tests/test_dl1eventsource.py

```diff
@@ -68,7 +68,9 @@ def test_simulation_info(dl1_file):
             for tel in event.simulation.tel:
                 assert tel in event.simulation.tel
                 assert event.simulation.tel[tel].true_image is not None
-                assert event.simulation.tel[tel].true_parameters.hillas.x != np.nan
+                assert (
+                    event.simulation.tel[tel].true_parameters.hillas.fov_lon != np.nan
```

Comment:
np.isnan, you cannot compare to nan (it will always be false).

Reply:
This is actually how its done on the master, good thing you spotted it now